### ISSUE TITLE
Can't set a placeholder in a CTkEntry when textvariable is enabled

### ISSUE THREAD

**Issue by Xeway** [TomSchimansky/CustomTkinter#917]
Hey,

I'm facing an issue when using `CTkEntry`.
``` python3
size_limit = StringVar()
size_limit.trace_add("write", size_limit_modified)

size_limit_entry = customtkinter.CTkEntry(master=split_tab,
                                          width=200,
                                          textvariable=size_limit,
                                          validate="key",
                                          placeholder_text="1GB = 1000MB = 1000000000B")
size_limit_entry.pack()
```
On this example, the placeholder doesn't show.
If I remove `textvariable`, it will work.

I don't know if this a bug or a problem in my code.

Thanks for your help.

**Comment by djnizzi** [comment]
same here

**Comment by kennarttech** [comment]
Yes is true i'm also facing the same problem though it has been documented that StringVar doesn't work with placeholder_text., 
i suggest Mr TomSchimansky work on this for us. with my humble request i plead if he can work on this for us

**Comment by shyshu02** [comment]
Here is a solution that can be used for reference: https://stackoverflow.com/questions/74973450/why-is-the-custom-tkinter-placeholder-text-for-an-entry-box-ignored

**Comment by FedericoSpada** [comment]
Hi all,
I'm not sure that showing the placeholder is the correct thing to do. If a Variable is associated, you always want to show the current value inside the variable, and so "" is displayed.
